lib/mocha/parameter_matchers/instance_methods.rb

```diff
@@ -16,3 +17,11 @@ def to_matcher
 class Object
   include Mocha::ParameterMatchers::InstanceMethods
 end
+
+# @private
+class Hash
```

Comment:
I think we can do without this monkey-patching, but will propose that for another PR.

Reply:
OK 👍